### ISSUE TITLE
what about more moderators ?

### ISSUE THREAD

**Issue by berak** [opencv-infrastructure/answers.opencv.org#10]
i'd propose LBerger, sturkmen, theoedore and pklab

**Comment by StevenPuttemans** [comment]
Moderators for the moment: me, berak, Michael Burdinov, Guanta, Mathieu Barnachon, Haris which is already quite the list right? But I am always up for more help.

Some insights on the suggested names
- LBerger & sturkmen --> good additions to the forum, but you notice that English is not their main language in their responses. So might not be the best options. On the other hand, the amount of responses coming from them is huge.
- Theodore LorenaGDL --> good choices, clear and concise in their answers and willing to help anyone
- pklab is Philip. I got the idea he is mostly active at the PR stuff, not sure if he is up to taking another job as a moderator. He is not that active on the forum.
- Uxnut might be a good choice, not noticed him that much lately. 
- Foobar indeed good addition, but he is one of the guys not being able to access his current account :D

**Comment by berak** [comment]
"but you notice that English is not their main language" - that applies to both of us, too ;)

**Comment by StevenPuttemans** [comment]
true ^_^ I actually wanted too write something like, the quality of English, you know what I mean :D

**Comment by StevenPuttemans** [comment]
@alekcac I do encourage you to set some new moderators! We could use them!

**Comment by alekcac** [comment]
Ok, I will look at them and their activity. I will add info here later.

**Comment by StevenPuttemans** [comment]
:+1:

**Comment by StevenPuttemans** [comment]
@alekcac as far as I know new moderators have been added so this can be closed down?

**Comment by PkLab** [comment]
@StevenPuttemans I read this just now 

> pklab is Philip

What do you mean ? I'm not so active with PR ?!? maybe some mistake 

In any way, thank you for your consideration  :smile:

**Comment by StevenPuttemans** [comment]
I mixed you up with Dikay I am afraid 😆

**Comment by berak** [comment]
we definitely need some fresh blood here.

as of now, only steven and me seem to be left , and if both of us stay away for a day or so [things pile up](http://answers.opencv.org/feeds/rss/?)

LorenaGDL, LBerger, Tetragram seem to be willing to fill the ranks, just assign some more folks, please !

**Comment by StevenPuttemans** [comment]
I agree.

Sorry for being at a conference for a week 😀. Not really the time to do my work from here 😋

**Comment by mshabunin** [comment]
@berak , I've added moderator rights to mentioned persons.

**Comment by berak** [comment]
thanks a lot !

**Comment by StevenPuttemans** [comment]
@mshabunin @berak thanks for getting more people to the club.
That being said, did anyone made them aware of their new powers?

**Comment by LorenaGdL** [comment]
@StevenPuttemans I guess so, that little warning sign appears too often not to notice it ;)
In any case, it would be interesting if you have any tips/guidelines or whatever to give us

**Comment by StevenPuttemans** [comment]
@LorenaGdL ;) true! Welcome to the club btw!

Well basically
1) Delete spam, but try to be attentive at new users posts to not delete and block them.
2) You can take breaks, if you do not feel like deleting, someone else will take over.
3) Use your common sense in moderating topics I would say. But basically if you get these powers, this means that current moderators like your style and approach. So just keep doing what you do.
4) Make sure if topics get locked or deleted, that you as moderator provide a clean explanation to the user as why this is happening.

**Comment by LaurentBerger** [comment]
@StevenPuttemans @LorenaGdL So do I. Now I apply following rule : 
1 block spammer 
2 accept new user if message is about opencv
3 reject message if it is not about opencv but it's not a spam (only one message) 

I think it's right.  
I made a dynamic cast < comment > answer in this post http://answers.opencv.org/question/114108/opencv-310-build-failure-on-windows-10/

**Comment by StevenPuttemans** [comment]
@LaurentBerger 

1) Yes block and delete content option
2) Yes
3) That's a bit difficult, I usually stick to 1 or 2 and let community decide if it is related or not

**Comment by LaurentBerger** [comment]
About 1-2-3 when I choose an option is it possible to go back?

**Comment by StevenPuttemans** [comment]
I do not think so. Decisions are quite final unless you remember the name of the user, then you can change his account status by looking for him through accounts.

**Comment by berak** [comment]
btw, nice to have you here, it makes such a difference ;)

**Comment by PkLab** [comment]
nice to see some fresh blood here. I Hope this might to improve some stale questions about the forum organization :-)

**Comment by sturkmen72** [comment]
hi all,
i am happy to became one of the moderators today. Hope together we contribute to a better forum environment.

**Comment by StevenPuttemans** [comment]
Welcome to the club!
